### PR TITLE
Libraries upgrade

### PR DESCRIPTION
**Pull request by Kamill90**

### CODE REVIEW

**Comment by KedziaPawel** [comment]
can we make a build from it and make a regression before merging?

**Comment by Kamill90** [comment]
> can we make a build from it and make a regression before merging?

Ask testers.

**Comment by KedziaPawel** [comment]
> > can we make a build from it and make a regression before merging?
> 
> Ask testers.

please prepare a build from the branch and tell Mateusz Marciniak to make a regression after it will be built

File: package.json [hunk 1]
```diff
@@ -43,10 +42,10 @@
     "flow-bin": "^0.102.0",
     "husky": "^4.2.3",
     "jail-monkey": "^2.3.2",
-    "jest": "^26.5.3",
+    "jest": "^25.1.0",
```
Comment (KedziaPawel) [inline package.json:45]: why we are downgrading jest?

Comment (Kamill90) [reply package.json]: Not any more

**Review by KedziaPawel — APPROVED** [review]

**Pull request merged**